src/algorithms/decentralized.md

```diff
@@ -35,8 +35,6 @@ The communication overhead of decentralized SGD is highly related to the degree
 ## Example
 
 
-## Benchmark
```

Comment:
we do need links to benchmark , for example

when xxxx, decentralized algorithm can be much faster than centralized counterparts, see xxxx

More evaluation results on decentralized algorithms can be found in xxxx paper

Reply:
We can refer to the benchmark page. For paper results, you mean Bagua paper? I think we don't need to cite algorithm papers in the benchmark section because these papers are mainly for the analysis of algorithms instead of benchmarks.